armi/settings/tests/test_settingsIO.py

```diff
@@ -45,15 +45,15 @@ def test_loadFromXmlFailsOnBadNames(self):
         with self.assertRaises(TypeError):
             ss.loadFromInputFile(None)
         with self.assertRaises(IOError):
-            ss.loadFromInputFile("this-settings-file-does-not-exist.xml")
+            ss.loadFromInputFile("this-settings-file-does-not-exist.yaml")
```

Comment:
I notice that even though you changed the file extension here, the name of the test that this falls within still refers to XML. I think either that name should be changed, or maybe this entire test can be removed?